todo_app/data/trello_items.py

```diff
@@ -1,22 +1,19 @@
-#from urllib import response
-#from cgitb import reset
-#from urllib import response
-#from urllib import response
 import requests
 import json
 import os
 from dotenv import load_dotenv
-
+load_dotenv()
+trello_key=(os.getenv('TRELLO_KEY'))
+trello_token=(os.getenv('TRELLO_TOKEN'))
+#Get Trello items
 def get_items():
    
   url = "https://api.trello.com/1/boards/62f2b96de0b99c00aff27ae6/lists?"
```

Comment:
You're hardcoding your trello board id here, but you should get it from an environment variable instead

Reply:
Created environment variable called TRELLO_BOARD_URL to resolve this